gfx/portraits/portrait_modifiers/50_beards_scripted_characters.txt

```diff
@@ -38,6 +38,16 @@ beards_scripted_characters = {
                 add = 200
 				is_nathanos_trigger = yes
             }
+            modifier = {
+                add = 200
+				exists = character:570000 # Garith Greenwood (Easter Egg)
+				this = character:570000 # Garith Greenwood (Easter Egg)
```

Comment:
Triggers should be used in other places too.
```suggestion
				is_garith_trigger = yes
```

Reply:
Fixed in 7b2b72e109bcdea81d141edeb6ad82bdba68f8e4. Also added triggers for **Aliza Colton** and **Ion of Hasic**, which were missing.